Tests/PerformanceTests.swift

```diff
@@ -0,0 +1,124 @@
+// Created by Cal Stephens on 2/14/22.
+// Copyright © 2022 Airbnb Inc. All rights reserved.
+
+import Foundation
+import XCTest
+
+@testable import Lottie
+
+// MARK: - PerformanceTests
+
+final class PerformanceTests: XCTestCase {
+
+  // MARK: Internal
+
+  func testAnimationViewSetup_simpleAnimation() {
+    // Compare the performance of displaying this simple animation in the two animation engines
+    let ratio = compareEngineSetupPerformance(for: simpleAnimation, iterations: 2000)
+
+    // This is basically a snapshot test for the performance of the Core Animation engine
+    // compared to the Main Thread engine. Currently, the Core Animation engine is
+    // about 1.6x slower than the Main Thread engine in this example.
+    XCTAssertEqual(ratio, 1.6, accuracy: 0.3)
+  }
+
+  func testAnimationViewSetup_complexAnimation() {
+    let ratio = compareEngineSetupPerformance(for: complexAnimation, iterations: 500)
+    XCTAssertEqual(ratio, 1.6, accuracy: 0.3)
+  }
+
+  func testAnimationViewScrubbing_simpleAnimation() {
+    let ratio = compareEngineScrubbingPerformance(for: simpleAnimation, iterations: 1000)
+    XCTAssertEqual(ratio, 0.1, accuracy: 0.1)
+  }
+
+  func testAnimationViewScrubbing_complexAnimation() {
+    let ratio = compareEngineScrubbingPerformance(for: complexAnimation, iterations: 1000)
+    XCTAssertEqual(ratio, 0.1, accuracy: 0.1)
+  }
+
+  // MARK: Private
+
+  private let simpleAnimation = Animation.named(
+    "loading_dots_1",
+    bundle: .module,
+    subdirectory: "Samples/LottieFiles")!
+
+  private let complexAnimation = Animation.named(
+    "LottieLogo2",
+    bundle: .module,
+    subdirectory: "Samples")!
+
+  /// Compares initializing the given animation with both the Main Thread and Core Animation engine,
+  /// and returns the ratio of how much slower the Core Animation is than the Main Thread engine
+  private func compareEngineSetupPerformance(for animation: Animation, iterations: Int) -> Double {
+    let mainThreadEnginePerformance = measurePerformance {
+      for _ in 0 ..< iterations {
```

Comment:
I believe our style guide recommends not having a space with range operators
```suggestion
      for _ in 0..<iterations {
```